toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -457,6 +468,13 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 		}
 	}
 
+	if ic.outputArtifactsDir != "" {
+		err = outputArtifacts(ic.config.Output.Artifacts.Items, ic.outputArtifactsDir, ic.buildDirAbs, ic.rawImageFile)
```

Comment:
Paths in the config file should be relative to the config file's parent directory, not the working directory.